core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -55,6 +55,8 @@
     boolean hash = false;
     @Parameter(names = {"--histogram"}, description = "print a histogram of the key-value sizes")
     boolean histogram = false;
+    @Parameter(names = {"--printIndex"}, description = "prints information about all the index entries")
+    boolean printIndex = false;
```

Comment:
Given some of the other discussion, I was thinking that a `--maxDepth` option might also be nice to add. Food for thought.